flow/peerdbenv/env.go

```diff
@@ -1,11 +1,17 @@
 package peerdbenv
 
 import (
+	"encoding/base64"
 	"encoding/json"
+	"fmt"
 	"os"
 	"reflect"
 	"strconv"
+	"strings"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/kms"
```

Comment:
Need to switch to `aws-sdk-go-v2` for pod identity to work mostly

Reply:
done